.env.production

```diff
@@ -3,4 +3,5 @@ VITE_TRACTION_GRAPHQL_URL=REPLACE_VITE_TRACTION_GRAPHQL_URL
 VITE_PRINTMYBARCODE_BASE_URL=REPLACE_VITE_PRINTMYBARCODE_BASE_URL
 VITE_SAMPLEEXTRACTION_BASE_URL=REPLACE_VITE_SAMPLEEXTRACTION_BASE_URL
 VITE_SEQUENCESCAPE_BASE_URL=REPLACE_VITE_SEQUENCESCAPE_BASE_URL
-VITE_SEQUENCESCAPE_API_KEY=REPLACE_VITE_SEQUENCESCAPE_API_KEY 
\ No newline at end of file
+VITE_SEQUENCESCAPE_API_KEY=REPLACE_VITE_SEQUENCESCAPE_API_KEY
+VITE_ENVIRONMENT=production
```

Comment:
This needs to be VITE_ENVIRONMENT=REPLACE_VITE_ENVIRONMENT and then REPLACE_VITE_ENVIRONMENT=production in the deployment project.

Reply:
Do I need to make a branch in deployment project for this change?